src/emulator/functionsEmulator.ts

```diff
@@ -647,6 +675,44 @@ export class FunctionsEmulator implements EmulatorInstance {
     }
   }
 
+  performPostLoadOperations(): Promise<void> {
```

Comment:
nit* I think this function can be simplified if it were written as `async` function -no need to deal with promises or promise chains.

Reply:
Sure, fixed